### PR TITLE
AA DS, on documentation, I see a recipe on how to set metadata for OCR

### PR DESCRIPTION
**Pull request by PaulKili**
- [x] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request

### CODE REVIEW

File: recipes/OCR_pre_annotations.ipynb [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def detect_text(path):\n",
+    "    \"\"\"Detects text in the file.\"\"\"\n",
+    "    from google.cloud import vision\n",
```
Comment (marcenacp) [inline recipes/OCR_pre_annotations.ipynb:222]: Imports outside of function

File: recipes/OCR_pre_annotations.ipynb [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+    "               \"boundingPoly\": {\n",
+    "                      \"vertices\": vertices,\n",
+    "                  },\n",
+    "              }\n",
+    "        \n",
+    "        textAnnotations.append(tmp)\n",
+    "        \n",
+    "    return textAnnotations\n",
+    "\n",
+    "    if response.error.message:\n",
```
Comment (marcenacp) [inline recipes/OCR_pre_annotations.ipynb:251]: This code cannot be reached. Put it above the `return`

Comment (PaulKili) [reply recipes/OCR_pre_annotations.ipynb:250]: @marcenacp done

File: recipes/OCR_pre_annotations.ipynb (hunk 1 above)
Comment (PaulKili) [reply recipes/OCR_pre_annotations.ipynb]: @marcenacp done

**Review by kili-technology — APPROVED** [review]

**Pull request merged**